dcp_prototype/backend/browser/api/app.py

```diff
@@ -0,0 +1,151 @@
+            'cell_count': project.cell_count,
+        })
+
+    return chalice.Response(status_code=200,
+                            headers={
+                                'Content-Type': "application/json",
+                                'Access-Control-Allow-Origin': "*"
+                            },
+                            body=projects)
+
+
+@app.route('/projects/{project_id}')
+def get_project(project_id):
+    session = DBSessionMaker().session()
+    project = session.query(Project).get(project_id)
+    response_body = {
+        'id': project.id,
+        'title': project.title,
+        'label': project.label,
+        'assays': _get_project_assays(session, project.id),
+        'tissues': _get_project_tissues(session, project.id),
+        'species': _get_project_species(session, project.id),
+        'description': project.description,
+        'category': project.category,
+        'developmental_stage': project.developmental_stage,
+        'disease_ontology': project.disease_ontology,
+        'sample_type': project.sample_type,
+        'organ_part': project.organ_part,
+        'analysis_protocol': project.analysis_protocol.split(','),
+        'cell_count': project.cell_count,
+        'donor_count': project.donor_count,
+        'publication_title': project.publication_title,
+        'publication_doi': project.publication_doi,
+        'contact_name': project.contact_name,
+        'contact_institution': project.contact_institution,
+        'contact_email': project.contact_email
+    }
+
+    return chalice.Response(status_code=200,
+                            headers={
+                                'Content-Type': "application/json",
+                                'Access-Control-Allow-Origin': "*"
+                            },
+                            body=response_body)
+
+
+@app.route('/projects/{project_id}/files')
+def get_project_files(project_id):
+    session = DBSessionMaker().session()
+    files = []
+    for file in session.query(File).filter(File.project_id == project_id).limit(100):
+        files.append({
+            'id': file.id,
+            'filename': file.filename,
+            'file_format': file.file_format,
+            'file_size': file.file_size,
+            'species': file.species,
+            'library_construction_method_ontology':
+                file.library_construction_method_ontology,
+            'tissue_ontology': file.tissue_ontology
+        })
+
+    return chalice.Response(status_code=200,
+                            headers={
+                                'Content-Type': "application/json",
+                                'Access-Control-Allow-Origin': "*"
+                            },
+                            body=files)
+
+
+@app.route('/files/{file_id}')
+def get_file(file_id):
+    session = DBSessionMaker().session()
+    file = session.query(File).get(file_id)
+    response_body = {
+        's3_uri': file.s3_uri
```

Comment:
Is this the same s3 uri from the ledger or are you moving the files into your own s3 bucket? The ledger bucket is private so files are not downloadable directly via s3 uri

Reply:
Yea, this endpoint is incomplete since file download was de-scoped for 2/21. Currently it's mainly a stub and will be implemented [next sprint](https://app.zenhub.com/workspaces/dcp2-5e2a191dad828d52cc78b028/issues/humancellatlas/dcp-prototype/131).

Since the API is only consumed by the frontend at the moment, leaving this stub as is.